common/src/auth.rs

```diff
@@ -153,7 +148,7 @@ impl SecretKey {
 
 impl PartialEq for SecretKey {
     fn eq(&self, other: &SecretKey) -> bool {
-        self.inner.0[..] == other.inner.0[..]
+        &self.inner.to_bytes()[..] == &other.inner.to_bytes()[..]
```

Comment:
Clippy error. Needs to be shortened to `self.inner.to_bytes() == other.inner.to_bytes()`.